### ISSUE TITLE
3.2.5 Change on Request (Level AAA)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#562]
#### [3.2.5 Change on Request](https://www.w3.org/TR/WCAG22/#change-on-request)

(Level AAA)

>[Changes of context](https://www.w3.org/TR/WCAG22/#dfn-change-of-context) are initiated only by user request or a [mechanism](https://www.w3.org/TR/WCAG22/#dfn-mechanism) is available to turn off such changes.

##### Applying SC 3.2.5 Change on Request to Non-Web Documents and Software
This applies directly as written, and as described in [Intent from Understanding Success Criterion 3.2.5](https://www.w3.org/WAI/WCAG22/Understanding/change-on-request.html#intent).

> **Note:** @@ Are notes needed? @@

***
**For WCAG2ICT consideration:** 
- Consider what notes might be needed for non-web software and documents
- Are there considerations for closed functionality?
***

**Comment by bruce-usab** [comment]
First impressions:
- Will be similar to AA SC [3.2.1 On Focus](https://w3c.github.io/wcag2ict/#on-focus), [3.2.2 On Input](https://w3c.github.io/wcag2ict/#on-input), and/or [4.1.3 Status Messages](https://w3c.github.io/wcag2ict/#status-messages).  (These three SC reference Change of Context).
- Understanding Note to Intent is _very_ web centric:
> It is possible for more than one change of context to occur simultaneously. For example, clicking on a link which opens a new window or tab is an example of two separate changes of context related to the change in content and to the change in the viewport (window or tab). The change in the content in this case is initiated by user request when they click on the link, but opening a new window or tab cannot be regarded as user-initiated if it happens without the user explicitly requesting it.

**Comment by bruce-usab** [comment]
**3.2.1** This applies directly as written, and as described in Understanding.

Note (added)
> Some compound documents and their user agents are designed to provide significantly different viewing and editing functionality depending upon what portion of the compound document is being interacted with (e.g. a presentation that contains an embedded spreadsheet, where the menus and toolbars of the user agent change depending upon whether the user is interacting with the presentation content, or the embedded spreadsheet content). If the user uses a mechanism other than putting focus on that portion of the compound document with which they mean to interact (e.g. by a menu choice or special keyboard gesture), any resulting [change of context](https://w3c.github.io/wcag2ict/#dfn-change-of-context) wouldn't be subject to this success criterion because it was not caused by a change of focus.

- Should this note be documents only?
- What is a “compound document”?

**3.2.3** This applies directly as written, and as described in Understanding.

No additional notes.

**3.2.5** This applies directly as written, and as described in Understanding.

NOTE 1 (ADDED)
> For [non-web documents](https://w3c.github.io/wcag2ict/#document) and [non-web software](https://w3c.github.io/wcag2ict/#software) where status messages are not implemented using markup languages, there is still a user need to have status messages be programmatically exposed so that they can be presented to the user by assistive technologies without receiving focus. This is typically enabled through the use of accessibility services of the [user agent](https://w3c.github.io/wcag2ict/#user-agent) or other [platform software](https://w3c.github.io/wcag2ict/#platform-software).

NOTE 2 (ADDED) (FOR NON-WEB SOFTWARE)
> See also the [Comments on Closed Functionality](https://w3c.github.io/wcag2ict/#comments-on-closed-functionality).

**Comment by bruce-usab** [comment]
TLDR:  After much over-thinking, applies as written and described in intent from Understanding.

See proposed [files changed](https://github.com/w3c/wcag2ict/pull/919/changes?w=1) from PR or  [diff preview](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F&doc2=https%3A%2F%2Fdeploy-preview-919--wcag2ict.netlify.app#change-on-request) at 3.2.5

- I do not think we need notes for either for non-web software or non-web documents.
- I do not believe that there are any considerations for closed functionality.

**Comment by pday1** [comment]
Thanks Bruce. Just to double check, your proposal is therefore as follows. Is that correct, @bruce-usab?

Applying SC 3.2.5 Change on Request to non-web documents and non-web software

This applies directly as written, and as described in [Intent from Understanding Success Criterion 3.2.5](https://www.w3.org/WAI/WCAG22/Understanding/change-on-request.html#intent).

**Comment by pday1** [comment]
In PR https://github.com/w3c/wcag2ict/pull/919. 
Now merged
